src/shared/cryptsetup-util.h

```diff
@@ -69,6 +70,9 @@ int dlopen_cryptsetup(void);
 DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(struct crypt_device *, crypt_free, NULL);
 DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(struct crypt_device *, sym_crypt_free, NULL);
 
+#define sym_crypt_free_and_replace(a, b)                \
```

Comment:
Shouldn't we just call this `crypt_free_and_replace()`? I don't think the fact that the library is dlopened should leak into macros and functions using it.

Reply:
OK. Now the suggested name is used.